### DISCUSSION TITLE
Error while testing NJOY

### DISCUSSION THREAD

**Discussion by Shatha-Elyazori** [njoy/NJOY21#187] [Q&A]
Newbie here! Just installed NJOY by following this documentation "https://docs.njoy21.io/install.html"  and when I try to test it,  I keep getting this error message: 'Errors while running CTest. Output from these tests are in: /home/ubuntu/NJOY21/bin/Testing/Temporary/LastTest.log. Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely. make: *** [Makefile:71: test] Error 8'. 
Any idea what might be causing this and how I can fix it? Thanks in advance for any help!

**Comment by whaeck** [comment]
NJOY21 is known to fail on a few of the tests depending on the system and compilers you are using. I cannot give you any advice as to what is going on with the limited amount of information that you provided here. I have seen tests fail on Mac M1 and M2 machines that compile in Release mode (Debug mode seems to be fine) due to roundoff issues in the test results (we do a comparison of the test files to verify success).

However, you can also try to use NJOY2016 directly (NJOY21 uses NJOY2016 under the hood): https://github.com/njoy/NJOY2016
The installation procedure for this version of NJOY is essentially the same as what you done for NJOY21 but it will go a lot faster (it only requires a Fortran compiler). Since you have managed to compile NJOY21, you should have no issue with NJOY2016.
